### PR TITLE
VSIX: Add a collection to visibility converter to hide tabs which do not contain data

### PR DESCRIPTION
**Pull request by rtaket**
Add a readme to explain the debugging settings for the VSIX project.

@michaelcfanning

### CODE REVIEW

File: src/Sarif.Viewer.VisualStudio/Converters/CollectionToVisiblityConverter.cs [hunk 1]
```diff
@@ -0,0 +1,33 @@
+﻿using System;
```
Comment (michaelcfanning) [inline src/Sarif.Viewer.VisualStudio/Converters/CollectionToVisiblityConverter.cs]: Add open source license header to this file?

File: src/Sarif.Viewer.VisualStudio/readme.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+﻿How to Debug
+============
+
+Set the following values in the Sarif.Viewer.VisualStudio project's Degug property page
```
Comment (michaelcfanning) [inline src/Sarif.Viewer.VisualStudio/readme.md]: Degug -> Debug

**Comment by michaelcfanning** [comment]
+1 with some minor suggestions

File: src/Sarif.Viewer.VisualStudio/Converters/CollectionToVisiblityConverter.cs (hunk 1 above)
Comment (rtaket) [inline src/Sarif.Viewer.VisualStudio/Converters/CollectionToVisiblityConverter.cs]: I didn't add the header to any of the new files. Is it required? (I can do them all in another PR.)

Comment (lgolding) [inline src/Sarif.Viewer.VisualStudio/Converters/CollectionToVisiblityConverter.cs]: Yes, it's required for code we put in a public repo.

**Comment by lgolding** [comment]
:+1:

**Pull request merged**